docs/user_needs/software_developer.sdoc

```diff
@@ -0,0 +1,23 @@
+[DOCUMENT]
+TITLE: Software Developer
+REQ_PREFIX: ZEP-
+
+[GRAMMAR]
+IMPORT_FROM_FILE: user_needs.sgra
+
+[FREETEXT]
+
+TBD
+
+[/FREETEXT]
+
+[USER_NEED]
+STATUS: Draft
+TYPE: User Need
+USER: Software Developer
+TITLE: Integrate a Semaphore Synchronization Primitive
+STATEMENT: >>>
+As a Software Developer threaded with implementing resource management in our real-time operating system (RTOS),
```

Comment:
I am not sure if this specific user need is formulated well enough to serve as a real Developer need. What exactly it this trying to say from the software developer perspective? Is the word `threaded` referring to something like `tasked with`?

Reply:
Both the of the STATEMENTS in this PR were copied from USER_STORY texts in the current systems_requirements.sdoc. I don't expect any of those USER_STORY statements have been scrutinized, so I am comfortable copying them over as Draft. (BTW, I have no idea what "threaded" means here. I just replicated the existing text.)

Reply:
Removing Software Developer section from this PR.